inc/saivlan.h

```diff
@@ -117,6 +130,15 @@ typedef enum _sai_vlan_attr_t
      */
     SAI_VLAN_ATTR_VLAN_ID = SAI_VLAN_ATTR_START,
 
+    /**
+     * @brief Vlan Type
+     *
+     * @type sai_vlan_type_t
+     * @flags CREATE_ONLY
+     * @default SAI_VLAN_TYPE_DATA
+     */
+    SAI_VLAN_ATTR_VLAN_TYPE,
```

Comment:
can you move this as last attribute? in ongoing trend to be enum values backward compatible this would break